src/types/user.ts

```diff
@@ -365,9 +366,9 @@ export default /* GraphQL */ `
     circleNewFollower: Boolean!
     circleNewUnsubscriber: Boolean!
     circleNewDiscussion: Boolean!
-    circleMemberBoradcast: Boolean!
-    circleMemberNewDiscussion: Boolean!
-    circleMemberNewDiscussionReply: Boolean!
+    circleMemberBoradcast: Boolean! # deprecated
```

Comment:
use directive `@deprecated(reason: "No longer in use")`?

Reply:
resolved in a80c6586